frontend/src/pages/ContactUs/ContactUs.jsx

```diff
@@ -3,9 +3,18 @@ import { Button2 } from "../../components/util/Button/index";
 import Recaptcha from "react-recaptcha";
 import style from "./ContactUs.module.scss";
 import Joi from "joi-browser";
+import { END_POINT } from "../../config/api";
```

Comment:
Please use the caselCase here i.e `endPoint` instead of `END_POINT`

Reply:
I can't because it is exported by that name only and changing it, I'll then have to make changes everywhere where it is used.